compliance/collection/main.go

```diff
@@ -289,15 +281,23 @@ func main() {
 	if features.RHCOSNodeScanning.Enabled() {
 		rescanInterval := env.NodeRescanInterval.DurationSetting()
 		cmetrics.ObserveRescanInterval(rescanInterval, getNode())
-		log.Infof("Node Rescan interval: %s", rescanInterval.String())
+		log.Infof("Node Scanning active and configured with rescan interval %s and cache duration %s", rescanInterval.String(), env.NodeScanCacheDuration.DurationSetting())
 
+		// TODO(ROX-13935): Remove FakeNodeInventory and its FF
 		var scanner nodeinventorizer.NodeInventorizer
 		if features.UseFakeNodeInventory.Enabled() {
 			log.Infof("Using FakeNodeInventorizer")
 			scanner = &nodeinventorizer.FakeNodeInventorizer{}
```

Comment:
Should this be now called `nodeinventorizer.FakeNodeAnalyzer{}`?

Reply:
That's a very good question. I'm still not happy with all the names. We now have a `NodeAnalyzer`, a `NodeInventorizer` and a `CachingScanner`, which all do very similar things but are named completely different.
Some unification is definitely needed in the component naming